DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/FlagRush_UI.Script.txt

```diff
@@ -27,6 +28,7 @@
 #Const C_LayerName_Feed						"Layer_FlagRush_Feed"
 #Const C_LayerName_Flash					"Layer_FlagRush_Flash"
 #Const C_LayerName_Spectator			"Layer_FlagRush_Spectator"
+#Const C_LayerName_Respawn				"Layer_FlagRush_Respawn"
```

Comment:
Should respawn and flash modules and layers be moerged to one? They are both respawn specific; Respawn shoing a timer and Flash showing the restriction message. @reaby also tell us what you think